plugins/output/adapt/importsource.js

```diff
@@ -86,9 +87,13 @@ function ImportSource(req, done) {
       async.apply(installPlugins),
       async.apply(addAssets, formTags),
       async.apply(cacheMetadata),
-      async.apply(importContent, formTags),
-      async.apply(helpers.cleanUpImport, cleanupDirs)
-    ], done);
+      async.apply(importContent, formTags)
+    ], function(importError, result) {
+      helpers.cleanUpImport(cleanupDirs, function(cleanUpError) {
```

Comment:
Can we get a comment here stating that `cleanUpImport` should be called regardless of whether the control function passes or fails - my brain couldn't process this 😂